janusgraph-solr/pom.xml

```diff
@@ -250,7 +282,7 @@
                             <systemProperties>
                                 <property>
                                     <name>index.search.solr.zookeeper-url</name>
-                                    <value>0.0.0.0:2181</value>
+                                    <value>172.17.0.1:9983</value>
```

Comment:
This address is the docker default gateway address. It would like to get rid of it. Beside this PR i also done some work on testcontainers which would cleanup this part.